File: 程式語言 V2答案檔/C++/409.cpp
#include <iostream>
#include <fstream>
using namespace std;

int main()
{
    fstream input, output;
    char c;
    int i;

    input.open("read.txt", ios::in);
    output.open("write.txt", ios::out);
    if (input.fail() || output.fail()) {
        cout << "檔案開啟失敗";
        return(0);
    }

    while (input.get(c)) {
        if (c >= 'a' && c <= 'z') {
            i = (int)(c + 2) - 'a';
            c = (char)((i % 26) + 'a');
            output << c;
        }
        else {
            output << c;
        }
    }
    input.close();
    output.close();
    return (0);
}

File: 程式語言 V2答案檔/PHP/409.php
<?php
    $file = fopen("read.txt", "r") or die("Can't open the file.");
    $str = fgets($file);
    $result = "";
    for($i = 0; $i < strlen($str); $i++){
        $result .= chr(ord($str[$i]) + 2);
    }
    $result_file = fopen("write.txt", "w") or die("Can't create the file");
    fwrite($result_file, $result);
    fclose($result_file);
?>